log/default.go

```diff
@@ -188,7 +188,11 @@ func (l *defaultLogger) Panicw(msg string, keysAndValues ...interface{}) {
 }
 
 func (l *defaultLogger) WithContext(_ context.Context) CommonLogger {
-	return l
+	return &defaultLogger{
+		stdlog: l.stdlog,
```

Comment:
Are there any reason that you initialize new struct instead of adjusting the old one's depth field? Is it for usecases we use WithContext and without context together?

Reply:
yes. The `WithContext` record is in the wrong place, so it needs to be adjusted.